app/helpers/host_helper/textual_summary.rb

```diff
@@ -198,6 +198,28 @@ def textual_network
     h
   end
 
+  def textual_cloud_networks
+    num   = @record.number_of(:cloud_networks)
```

Comment:
Does `Host` really have `cloud_networks` ?

Reply:
It might sound strange, but it has:
```
irb(main):005:0> Host.last.cloud_networks
  Host Load (1.0ms)  SELECT  "hosts".* FROM "hosts" ORDER BY "hosts"."id" DESC LIMIT $1  [["LIMIT", 1]]
  Host Inst Including Associations (0.4ms - 1rows)
  CloudNetwork Load (2.1ms)  SELECT "cloud_networks".* FROM "cloud_networks" INNER JOIN "cloud_subnets" ON "cloud_networks"."id" = "cloud_subnets"."cloud_network_id" INNER JOIN "cloud_subnets_network_ports" ON "cloud_subnets"."id" = "cloud_subnets_network_ports"."cloud_subnet_id" INNER JOIN "network_ports" ON "cloud_subnets_network_ports"."network_port_id" = "network_ports"."id" WHERE "network_ports"."device_id" = $1 AND "network_ports"."device_type" = $2  [["device_id", 11], ["device_type", "Host"]]
  CloudNetwork Inst Including Associations (0.4ms - 1rows)
=> #<ActiveRecord::Associations::CollectionProxy [#<ManageIQ::Providers::Openstack::NetworkManager::CloudNetwork::Private id: 16, name: "ctlplane", ems_ref: "0e9720ab-e4d3-4701-ab95-4142ae179fb9", ems_id: 26, cidr: nil, status: "active", enabled: true, external_facing: false, cloud_tenant_id: nil, orchestration_stack_id: nil, shared: false, provider_physical_network: "ctlplane", provider_network_type: "flat", provider_segmentation_id: nil, vlan_transparent: nil, extra_attributes: {:port_security_enabled=>nil, :qos_policy_id=>nil, :maximum_transmission_unit=>1500}, type: "ManageIQ::Providers::Openstack::NetworkManager::Cl...">]>

```

Reply:
What's the semantics of it then? Is it really a "Cloud Network" as this fix suggests? The word "Cloud" really confuses me here (seeing we're looking at a Host).